tests/MongoDB.Driver.Legacy.Tests/MongoServerSettingsTests.cs

```diff
@@ -113,6 +113,33 @@ public void TestClone()
             settings.SdamLogFilename = "unimatrix-zero";
 
             var clone = settings.Clone();
+
+            Assert.Equal(settings, clone);
+        }
+
+        [Fact]
+        public void TestCloneTlsDisableCertificateRevocationCheck()
+        {
+            var connectionString = "mongodb://somehost/?tlsDisableCertificateRevocationCheck=true";
+            var builder = new MongoUrlBuilder(connectionString);
+            var url = builder.ToMongoUrl();
+            var settings = MongoServerSettings.FromUrl(url);
+
+            var clone = settings.Clone();
+
+            Assert.Equal(settings, clone);
```

Comment:
use a new format for test asserting

Reply:
True in general. Sometimes in a file that uses the old style asserts it is acceptable to match the style in the file.

Reply:
Done.